mungegithub/submit-queue/deployment/contrib/configmap.yaml

```diff
@@ -3,41 +3,42 @@ kind: ConfigMap
 metadata:
   name: contrib-sq-config
 data:
-  # basic config options.
```

Comment:
I think that's a good idea. I don't want to change the Makefile as well in this PR, I'll check that approach in a subsequent one.